src/common/SlideAnimationView.js

```diff
@@ -46,7 +46,7 @@ export default class SlideAnimationView extends PureComponent<Props, State> {
     const { property, from, to, movement, style } = this.props;
     const animationValue = this.state.animationIndex.interpolate({
       inputRange: [0, 1],
-      outputRange: movement === 'out' ? [from, to] : [to, from],
+      outputRange: movement === 'out' ? ([from, to]: number[]) : ([to, from]: number[]),
```

Comment:
Could this annotation be applied once to the whole expression? Would make it a bit shorter.

Reply:
Yep! Thanks, done.